### DISCUSSION TITLE
Cannot verify

### DISCUSSION THREAD

**Discussion by ContentLTD** [community/community#142320] [Projects and Issues]
Hello, I am trying to log into GitHub on my PC, but when I do so, I goto verify with the mobile app. When I do so I get the error below.![image](https://github.com/user-attachments/assets/a629d584-9d2c-45ff-a7b9-bd551ec19f5c)

**Comment by ghost** [comment]
Have you got any other alternative 2FA method you could use for signing into GitHub on your computer like a phone number or a recovery code?

**Reply by ContentLTD** [reply]
No, I don't. My security key is not working with the computer i want to sign in on
